<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NMediaManager::NPreferences::NUi::CPathsToDelete</class>
 <widget class="QWidget" name="NMediaManager::NPreferences::NUi::CPathsToDelete">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>405</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QCheckBox" name="deleteEXE">
     <property name="text">
      <string>Delete Executables (*.exe)?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="deleteTXT">
     <property name="text">
      <string>Delete Text Files (*.txt)?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="deleteNFO">
     <property name="text">
      <string>Delete NFO Files (*.nfo)?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="deleteBAK">
     <property name="text">
      <string>Delete Backup Files (*.bak)?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="deleteImages">
     <property name="text">
      <string>Delete Images?</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="imageLabelExtensions">
     <property name="text">
      <string>Image Extensions:</string>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="deleteCustom">
     <property name="title">
      <string>Custom Paths to Delete</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_12">
      <item row="5" column="1">
       <widget class="QToolButton" name="btnDelPathToDelete">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/delete.png</normaloff>:/SABUtilsResources/delete.png</iconset>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <spacer name="verticalSpacer_11">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>174</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="4" column="1">
       <widget class="QToolButton" name="btnAddPathToDelete">
        <property name="text">
         <string>...</string>
        </property>
        <property name="icon">
         <iconset resource="../../SABUtils/resources/SABUtils.qrc">
          <normaloff>:/SABUtilsResources/add.png</normaloff>:/SABUtilsResources/add.png</iconset>
        </property>
       </widget>
      </item>
      <item row="4" column="0" rowspan="3">
       <widget class="QListView" name="pathsToDelete"/>
      </item>
      <item row="3" column="0" colspan="2">
       <widget class="QLabel" name="label_13">
        <property name="text">
         <string>glob expressions (*,? etc are acceptable)</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>deleteEXE</tabstop>
  <tabstop>deleteTXT</tabstop>
  <tabstop>deleteNFO</tabstop>
  <tabstop>deleteBAK</tabstop>
  <tabstop>deleteImages</tabstop>
  <tabstop>deleteCustom</tabstop>
  <tabstop>pathsToDelete</tabstop>
  <tabstop>btnAddPathToDelete</tabstop>
  <tabstop>btnDelPathToDelete</tabstop>
 </tabstops>
 <resources>
  <include location="../../SABUtils/resources/SABUtils.qrc"/>
 </resources>
 <connections/>
</ui>
